lib/eyeshade/wallet.rb

```diff
@@ -25,6 +27,8 @@ def initialize(wallet_json:, channel_json:)
       @possible_currencies = details_json["possibleCurrencies"] || []
       @address = details_json["address"] || ""
       @is_member = details_json["isMember"] || false
+      @status  = details_json["status"]
+      @uphold_id = details_json["id"]
```

Comment:
Do we know if it's `id` or `address`? @evq would you verify this?

Reply:
it's `id`. `address` is the card address of the first card that matches the user's preferred currency